xarray/tests/test_backends.py

```diff
@@ -2213,6 +2212,77 @@ class PyNioTestAutocloseTrue(PyNioTest):
     autoclose = True
 
 
+@requires_netCDF4
+@requires_pnc
+class TestPseudoNetCDF(CFEncodedDataTest, NetCDF3Only, TestCase):
+    def test_write_store(self):
+        # pseudonetcdf is read-only for now
+        pass
+
+    @contextlib.contextmanager
+    def open(self, path, **kwargs):
+        with open_dataset(path, engine='pseudonetcdf',
+                          autoclose=self.autoclose,
+                          **kwargs) as ds:
+            yield ds
+
+    def save(self, dataset, path, **kwargs):
+        dataset.to_netcdf(path, engine='netcdf4', **kwargs)
+
+    def test_weakrefs(self):
```

Comment:
We don't need this test -- it was specifically for some issue with pynio.

Reply:
Referring to test_weakrefs right? Done.